docs/rules/no-native-attributes.md

```diff
@@ -13,20 +13,24 @@ The following patterns are considered warnings:
 
 ```ts
 class MyEl extends LitElement {
-  static properties = {
-    title: { type: String },
-    role: { type: String },
-  };
+  static get properties() {
+    return {
+      title: { type: String },
+      role: { type: String },
```

Comment:
and the trailing comma to go too :eyes: (fwiw this is mostly so if i ever autoformat the md it'll already be consistent, but also just for sake of manual consistency)

Reply:
hrm, looks like whenever I save the file it re-adds the trailing comma :/ not sure if thats on my vs code configuration or the projects configuration?

Reply:
strange, the `.prettierrc` in the repo specifies `none` which is why i was suggesting to change this

maybe your local vscode isnt picking up the config?

Reply:
its weird, I dont think I even have the prettier extension installed